### PR TITLE
Zeroone

### PR DESCRIPTION
**Pull request by deadthings**
I have been testing this "zocified" Sentinel for about 2 weeks to make sure I didn't inadvertently break something. Everything works as it should. Note that I have not touched the unit tests, which will all need to be rewritten for ZOC.

### CODE REVIEW

File: share/travis_setup.sh [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 set -evx
 
-mkdir ~/.dashcore
+mkdir ~/.zeroonecore
 
 # safety check
-if [ ! -f ~/.dashcore/.dash.conf ]; then
-  cp share/dash.conf.example ~/.dashcore/dash.conf
-fi
+if [ ! -f ~/.zeroonecore/.zeroone.conf ]; then
```
Comment (camenlx) [inline share/travis_setup.sh]:
```suggestion
if [ ! -f ~/.zeroonecore/zeroone.conf ]; then
```

File: lib/zeroonelib.py [hunk 2]
```diff
@@ -12,14 +12,14 @@
...
-def is_valid_dash_address(address, network='mainnet'):
+def is_valid_zeroone_address(address, network='mainnet'):
     # Only public key addresses are allowed
     # A valid address is a RIPEMD-160 hash which contains 20 bytes
     # Prior to base58 encoding 1 version byte is prepended and
     # 4 checksum bytes are appended so the total number of
     # base58 encoded bytes should be 25.  This means the number of characters
     # in the encoding should be about 34 ( 25 * log2( 256 ) / log2( 58 ) ).
-    dash_version = 140 if network == 'testnet' else 76
+    zeroone_version = 140 if network == 'testnet' else 76
```
Comment (camenlx) [inline lib/zeroonelib.py]:
```suggestion
    zeroone_version = 112 if network == 'testnet' else 80
```

File: bin/dbtest.py [hunk 3]
```diff
@@ -9,17 +9,17 @@
...
+zerooned = ZeroOneDaemon.from_zeroone_conf(config.zeroone_conf)
 import misc
 # ==============================================================================
 # do stuff here
 
 pr = Proposal(
     name='proposal7',
-    url='https://dashcentral.com/proposal7',
+    url='https://zeroonecentral.com/proposal7',
     payment_address='yTC62huR4YQEPn9AJHjnQxxreHSbgAoatV',
```
Comment (camenlx) [inline bin/dbtest.py]:
```suggestion
    payment_address='nF3PndkR4vttZKUpYdYQjDCCrfsfhNGs6A',
```

File: bin/dbtest.py [hunk 4]
```diff
@@ -9,17 +9,17 @@
...
-dashd = DashDaemon.from_dash_conf(config.dash_conf)
+zerooned = ZeroOneDaemon.from_zeroone_conf(config.zeroone_conf)
 import misc
 # ==============================================================================
 # do stuff here
 
 pr = Proposal(
     name='proposal7',
-    url='https://dashcentral.com/proposal7',
+    url='https://zeroonecentral.com/proposal7',
```
Comment (camenlx) [inline bin/dbtest.py:22]: please also include ZOC testnet valid address, example:
nF3PndkR4vttZKUpYdYQjDCCrfsfhNGs6A

File: bin/dbtest.py (hunk 3 above)
Comment (camenlx) [inline bin/dbtest.py]:
```suggestion
    payment_address='nF3PndkR4vttZKUpYdYQjDCCrfsfhNGs6A',
```

File: bin/dbtest.py (hunk 4 above)
Comment (deadthings) [reply bin/dbtest.py:22]: Nice work - thanks for the review cmelx!

**Pull request merged**